### PR TITLE
Improve dark mode 

### PR DESCRIPTION
**Pull request by mohe2015**
by @JuliaSelua 

Closes https://github.com/reform-org/reform/issues/359
Closes https://github.com/reform-org/reform/issues/357

### CODE REVIEW

File: js/src/main/scala/webapp/entity/EntityPage.scala [hunk 1]
```diff
@@ -122,7 +122,7 @@ class EntityRow[T <: Entity[T]](
             }),
         ),
       td(
-        cls := "py-1 min-w-[185px] max-w-[185px] mx-auto sticky right-0 bg-white dark:bg-gray-700 border-x border-b border-gray-300 dark:border-gray-700 !z-[1]",
+        cls := "py min-w-[185px] max-w-[185px] mx-auto sticky right-0 bg-white dark:bg-gray-600 odd:bg-blue-600 border-x border-b border-gray-300 dark:border-gray-600 !z-[1]",
```
Comment (lukasschreiber) [inline js/src/main/scala/webapp/entity/EntityPage.scala:125]: why is this blue?

Comment (mohe2015) [reply js/src/main/scala/webapp/entity/EntityPage.scala:125]: @JuliaSelua

Comment (mohe2015) [reply js/src/main/scala/webapp/entity/EntityPage.scala:125]: When I looked at it nothing looked of so maybe it's just a really dark blue?

**Comment by mohe2015** [comment]
The input fields for username and password in the connection drawer look too light gray

File: js/src/main/scala/webapp/entity/EntityPage.scala (hunk 1 above)
Comment (lukasschreiber) [reply js/src/main/scala/webapp/entity/EntityPage.scala:125]: no it's not, 600 is the saturation we use for all the really vivid colors in the button, so it should be noticable, propably overwritten by some  rule in index.css, could just be removed

**Pull request merged**